### PR TITLE
Update/nft picking page

### PR DESCRIPTION
**Pull request by Amean**
Updated the page where nft's are picked for the home widget

### CODE REVIEW

File: wallet/lib/pages/home/home.dart [hunk 1]
```diff
@@ -59,6 +60,45 @@ class HomeScreenState extends State<HomeScreen> with SingleTickerProviderStateMi
...
+  void _launchedFromWidget(Uri? uri) {
+    if (uri!=null) {
+      Navigator.of(context).pushNamed(RouteUtil.ROUTE_WIDGET);
+    }
+  }
+
+
+
+
+  // Future<void> image_picker() async {
```
Comment (mustafapylons) [inline wallet/lib/pages/home/home.dart:83]: Will review this more once CI passes, but for now we can remove this dead code.

**Review by mustafapylons — COMMENTED** [review]
Will wait for CI to pass for further review.

**Review by faddat — APPROVED** [review]

File: wallet/lib/pages/home/collection_screen/home_widget.dart [hunk 2]
```diff
@@ -0,0 +1,295 @@
...
+    } on PlatformException catch (exception) {
+      debugPrint('Error Updating Widget. $exception');
+    }
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    final collectionViewModel = GetIt.I.get<CollectionViewModel>();
+    return
+    AnnotatedRegion<SystemUiOverlayStyle>(
```
Comment (kjawadDeveloper2) [inline wallet/lib/pages/home/collection_screen/home_widget.dart:95]: Kindly format this screen.

File: wallet/lib/pages/home/collection_screen/home_widget.dart [hunk 3]
```diff
@@ -0,0 +1,295 @@
...
+                    childrenDelegate: SliverChildBuilderDelegate((context, index) {
+                      final nft = viewModel.purchases[index];
+
+                      if (isSelected) {
+                        isSelected = false;
+                        return GestureDetector(
+                          onTap: () async {
+                            imagePicked = nft.url;
+                            isSelected = true;
+                            setState(() {});
```
Comment (kjawadDeveloper2) [inline wallet/lib/pages/home/collection_screen/home_widget.dart:138]: Kindly avoid set state.

File: wallet/lib/pages/home/collection_screen/home_widget.dart [hunk 4]
```diff
@@ -0,0 +1,295 @@
...
+          }
+      );
+    }
+    ),
+    ),
+    );
+  }
+}
+
+class PylonButtonClipper extends CustomClipper<Path> {
```
Comment (kjawadDeveloper2) [inline wallet/lib/pages/home/collection_screen/home_widget.dart:276]: I think we already has this clipper. Kindly reuse please.

**Comment by Tanyaml** [comment]
@kjawadDeveloper2 please review.

**Review by MikeSofaer — APPROVED** [review]

**Pull request closed without merging**